docker/oso-host-monitoring/src/Dockerfile.j2

```diff
@@ -78,6 +78,12 @@ RUN yum clean metadata && \
 
 ADD urllib3-connectionpool-patch /root/
 RUN yum install -y patch && yum clean all && cd /usr/lib/python2.7/site-packages/ && patch -p1 < /root/urllib3-connectionpool-patch
+{% if base_os == "centos7" %}
+
+# Add copr repo for python-hawkular-client rpm and install it
+RUN cd /etc/yum.repos.d && curl -O https://copr.fedorainfracloud.org/coprs/zohargalor/python-hawkular-client/repo/epel-7/zohargalor-python-hawkular-client-epel-7.repo
+RUN yum install -y python-hawkular-client
```

Comment:
In docker, you ALWAYS need to clean up after yum so that it's cached files aren't committed to the image layer.

So this line should be something like this (untested):
```
RUN yum install -y python-hawkular-client && yum clean all
```


Also, do we want to do this install only on centos?

I believe the package is available for rhel as well. 

The copr repo addition, however, should only be in the centos section.